package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "start": "vite dev",
-    "build": "npm run sync && vite build && ./bin/postbuild.js",
```

Comment:
Why did you remove this?

Reply:
My brain read `postcss` instead of `postbuild`, woops